src/models/competences.ts

```diff
@@ -29,6 +40,6 @@ export const competencesAutres = [
 
 export const competences = {
     Produit: competencesProduit,
-    Développement: competencesTechniques,
```

Comment:
Pas fan d'avoir en clés des valeurs qui sont liées à l'affichage (genre `Développement`), mais not a big deal, it works